docs/security/code-integrity.md

```diff
@@ -29,13 +29,13 @@ See the following documentation to learn more:
 
 [VBI](../boot/vbi.md)
 
-Prior to Windows 10/11 thresholds Windows Core 8.1 (Durango) based firmware relied heavily on standard CI protection, Xbox OS 10/11 based firmware now adds the addition of XCI (Xbox Code Integrity) to the existing CI chain to validate Xbox specific virtual images, catalogs, and CAB (Cabinet) files.
+Prior to Windows 10/11 thresholds, Windows Core 8.1 (Durango) based firmware relied heavily on standard CI protection. Xbox OS 10/11 based firmware now adds the addition of XCI (Xbox Code Integrity) to the existing CI chain to validate Xbox specific virtual images, catalogs, and CAB (Cabinet) files.
 
 In addition to Host, System, and GameOS VMs mounting read-only [XVDs](../operating-system/xbox-virtual-drive.md), effectively preventing writes and/or modifications to the file system, each virtual disk also contains the presence of catalog files in the root directory.
 
 ## Xbox Catalog Files
 
-Catalog files are generated in the root of every virtual disk on release build firmware and is checked by code integrity enforcement. The Binary Signatures contained within the Catalog files determine the conditions in which a binary matching the signature can execute into memory.
+Catalog files are generated in the root of every virtual disk on release build firmware,, and is checked by code integrity enforcement. The Binary Signatures contained within the Catalog files determine the conditions in which a binary matching the signature can execute into memory.
```

Comment:
Two commas slipped in here.

Reply:
Not sure how that happened LOL, will edit the commit